gremlin-go/driver/graphBinary.go

```diff
@@ -131,6 +135,20 @@ func listWriter(value interface{}, buffer *bytes.Buffer, typeSerializer *graphBi
 	return buffer.Bytes(), nil
 }
 
+// Format: {length}{value}
+func byteBufferWriter(value interface{}, buffer *bytes.Buffer, typeSerializer *graphBinaryTypeSerializer) ([]byte, error) {
+	v := value.(*ByteBuffer)
+	err := binary.Write(buffer, binary.BigEndian, int32(len(v.Data)))
+	if err != nil {
+		return nil, err
+	}
+	buffer.Write(v.Data)
+
+	return buffer.Bytes(), nil
+
+	return listWriter(v, buffer, typeSerializer)
```

Comment:
Something weird here.

Reply:
fixed in https://github.com/lyndonbauto/tinkerpop/pull/112/commits/86857c054213c55ffb3c0bc79468c8d158a4e1da